docs/en/index.md

```diff
@@ -1,48 +1,15 @@
-title: Login Forms
+---
+title: Login forms
 summary: Simplifying the login forms for a SilverStripe site
+icon: file-lines
+---
 
-# Introduction
+# Login forms
 
-The [Login Forms module](https://addons.silverstripe.org/add-ons/silverstripe/login-forms) provides an improved default
-view for the authentication UIs in SilverStripe. Rather than applying the forms to the site's default Page template,
-Login Forms renders a simple, responsive, standalone interface that aligns with the design of the CMS. This can be
-tweaked to suit an individual SilverStripe site, but doesn't require any extra work out of the box to incorporate the UI
-into the site's design.
+Thos module provides an improved default view for the authentication UIs in SilverStripe.
 
-# Providing a custom brand
+Rather than applying the forms to the site's default Page template, Login Forms renders a simple, responsive, standalone interface that aligns with the design of the CMS.
 
-The Login Forms module supports use of a custom brand that will replace the Site Name above the form. The best way to
-provide this is in SVG format, as one SVG can be used for all screen densities / display modes, whilst using images will
-require variations for each.
+This can be tweaked to suit an individual SilverStripe site, but doesn't require any extra work out of the box to incorporate the UI into the site's design.
```

Comment:
```suggestion
This can be tweaked to suit an individual Silverstripe CMS site, but doesn't require any extra work out of the box to incorporate the UI into the site's design.
```